examples/api/histogram_path.py

```diff
@@ -3,8 +3,8 @@
 Building histograms using Rectangles and PolyCollections
 ========================================================
 
-This example shows how to use a path patch to draw a bunch of
-rectangles.  The technique of using lots of Rectangle instances, or
+A path patch to draw a bunch of rectangles.
```

Comment:
Sentence fragment does not make sense.

Reply:
I think it does here

Reply:
It might have worked if it were the only thing, but right now, it's just a subject with no verb.

Reply:
Suggest:   Using a path patch to draw rectangles.